mlir/include/mlir/Dialect/Vector/Transforms/VectorRewritePatterns.h

```diff
@@ -235,6 +235,12 @@ void populateVectorExtractStridedSliceToExtractInsertChainPatterns(
     std::function<bool(ExtractStridedSliceOp)> controlFn = nullptr,
     PatternBenefit benefit = 1);
 
+/// Populate `patterns` with a pattern to rewrite simple cases of N-D
+/// extract_strided_slice, where the slice is contiguous, into extract and
```

Comment:
ubernit: I know this follows the existing convention in this file, but darn is this first part of the sentence redundant :D I'd say 'Patterns to rewrite ...'
(not requesting any change here, just venting...)

Reply:
done!